internal/pkg/template/templates/workloads/partials/cf/https-listener.yml

```diff
@@ -20,16 +20,17 @@ LoadBalancerDNSAlias:
       AliasTarget:
         HostedZoneId: !GetAtt EnvControllerAction.PublicLoadBalancerHostedZone
         DNSName: !GetAtt EnvControllerAction.PublicLoadBalancerDNSName
-{{- else if .HostedZoneID}}
-LoadBalancerDNSAlias:
+{{- else}}
+{{- range $hostedZoneID, $aliases := .HostedZone}}
```

Comment:
I'm probably missing something but doesn't this mean that _there has to be at least one_ `hosted_zone` field under `http`?

like what if I have:
```yaml
http:
  alias: [example.com, api.example.com]
``` 
Wouldn't this `{{range}}` block try to add the `defaultHostedZone := m.HostedZone` right now?

Reply:
If it is
```yaml
http:
  alias: [example.com, api.example.com]
```
No A record adding for them using the CFN resource.

if it is
```yaml
http:
  alias: [example.com, api.example.com]
  hosted_zone: ABCD123
```
then it applies to both `example.com` and `api.example.com`

Reply:
ok I see - in `convertHostedZone` if there is no `hosted_zone` field then we return an empty map that's not nil. But since the map is empty this range-block does nothing 👍 we're good!